sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/DigitalTwinMetadata.cs

```diff
@@ -25,7 +26,7 @@ public class BasicDigitalTwinMetadata
         /// This field will contain metadata about changes on properties on the digital twin.
         /// For your convenience, the value can be deserialized into <see cref="DigitalTwinPropertyMetadata"/>.
         /// </summary>
-        [JsonExtensionData]
-        public IDictionary<string, object> PropertyMetadata { get; set; } = new Dictionary<string, object>();
+        //[JsonExtensionData]
+        public IDictionary<string, DigitalTwinPropertyMetadata> PropertyMetadata { get; set; } = new Dictionary<string, DigitalTwinPropertyMetadata>();
```

Comment:
The impact to this change is that this dictionary can be `<string, DigitlaTwinPropertyMetadata>` instead of object, meaning the user won't have to cast it.